src/lib.rs

```diff
@@ -52,6 +66,41 @@ impl Offer {
         self.offer.description().map(|s| s.to_string())
     }
 
+    #[wasm_bindgen(getter)]
+    pub fn expiry(&self) -> Option<u64> {
+        self.offer.absolute_expiry().map(|d| d.as_secs())
+    }
+
+    #[wasm_bindgen(getter)]
+    pub fn issuer(&self) -> Option<String> {
+        self.offer.issuer().map(|s| s.to_string())
+    }
+
+    #[wasm_bindgen(getter)]
+    pub fn quantity(&self) -> Option<u64> {
+        match self.offer.supported_quantity() {
+            lightning::offers::offer::Quantity::Bounded(n) => Some(n.get()),
+            lightning::offers::offer::Quantity::Unbounded => Some(0),
```

Comment:
Not sure if 0 is a good way to communicate unbounded. How about returning `None`?